tools/ci-build.sh

```diff
@@ -150,7 +159,7 @@ function build_bmv2() {
     P4C_RUNTIME_DEPS+=" gcc-9 g++-9"
     export CC=gcc-9
     export CXX=g++-9
-  else
+  elif [[ "${DISTRIB_RELEASE}" != "24.04" ]] ; then
```

Comment:
Does this check for versions other than 24.04 mean that p4lang-bmv2 is not installed on Ubuntu 24.04 tests, and thus Ubuntu 24.40 CI does not run any tests that pass packets through bmv2?

If so, I guess the long term way to improve that is to create a way to build bmv2 binaries for p4c testing that run on Ubuntu 24.04?

Reply:
Yes there is no `p4lang-bmv2` for Ubuntu 24.04. We could build BMv2 from scratch, but that adds CI time.